ocaml-eglot-util.el

```diff
@@ -94,6 +110,12 @@ If optional MARKERS, make markers instead."
              (target (+ offset-l offset-c)))
         (byte-to-position target)))))
 
+(defun ocaml-eglot-util--pos-to-point (pos)
```

Comment:
You should probably use this at the other existing call-site of ocaml-eglot-util--point-by-pos in ocaml-eglot--phrase